turbine/src/sigverify_shreds.rs

```diff
@@ -125,14 +130,36 @@ fn run_shred_sigverify<const K: usize>(
     });
     verify_packets(
         thread_pool,
-        self_pubkey,
+        &keypair.pubkey(),
         bank_forks,
         leader_schedule_cache,
         recycler_cache,
         &mut packets,
         cache,
     );
     stats.num_discards_post += count_discards(&packets);
+    // Resign shreds Merkle root as the retransmitter node.
```

Comment:
Trying to recall the behavior we want for repair shred requests. Looks like we are signing repair requests here. Do we plan on verifying those signatures?

Reply:
I am not sure yet if we need to verify signature on repair shreds or just rely on verifying the nonce.

Nonetheless, resigning all shreds here makes my follow up changes a lot easier and since repair shreds are a very small percentage it shouldn't impact performance much.